app/src/main/res/values/strings.xml

```diff
@@ -38,6 +38,11 @@
     <string name="fmt_timer_start">Start</string>
     <string name="fmt_timer_pause">Pause</string>
     <string name="fmt_timer_reset">Reset</string>
+    <string name="fmt_timer_edit">Edit</string>
+    <string name="fmt_timer_confirm">Confirm</string>
+    <string name="fmt_timer_select_min">Min</string>
+    <string name="fmt_timer_select_sec">Sec</string>
+    <string name="fmt_timer_menu_change">Change Timer</string>
```

Comment:
Can you remove `fmt` here and change `menu_fmt` to "Timer"?

Reply:
yep on it